concierge_scheduler/concierge_zabbix.py

```diff
@@ -307,32 +380,96 @@ def import_actions(self):
             # "pyzabbix.ZabbixAPIException: ('Error -32500: Application error.,
             # No permissions to referred object or it does not exist!', -32500)"
             _warn(str(err))
+        _info('Importing trigger actions')
         self.import_trigger_actions()
+        _info('Importing registration actions')
         self.import_registration_actions()
 
-    def import_media_types(self, component):
+    def compare_ids(self, component):
+        """
+        Compare the ids and hashes of components for imported configuration.
+        Remove any components that are in the destination server but not in the imported configuration
+        :param component: Which component to compare the data of
         """
-        import backed up media
 
-        :param component: what media to import
-        :return:
+        component_id = self.id_mapping[component]['id']
+        ids_to_delete = []
+        for item in self.original_ids[component]:
+            import_id = item[component_id]
+            if import_id in self.dest_ids[component]:
+                if item["hash"] != self.dest_ids[component][import_id][1]:
```

Comment:
I'm not following how the hash of the original ID compares to import_id element 1. Please comment

Reply:
Originally `self.dest_ids` had a tuble with `name` and `hash` for each id.  so element 1 would be referring to the hash. 
```python
self.dest_ids = {templates: {10000: (name, hash), 10001:(name, hash}}
```
This has been updated to now have a map rather than a tuple.